src/components/UI/atoms/StyledLink/StyledLink.spec.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { render, cleanup } from '<src>/helpers/testUtils';
+import StyledLink from './StyledLink';
+import Title from '../Title/Title';
+import '@testing-library/jest-dom/extend-expect';
+
+afterEach(cleanup);
+
+describe('StyledLink', () => {
+  it('should render the Link component', () => {
+    const { getByText } = render(
+      <MemoryRouter>
```

Comment:
Please look at your colleagues' tests, I expect you component tests not to have repeated logic